### PR TITLE
Docs: General Metrics

### LINKED ISSUE #1280: Document Iroha metrics
At the moment the only way to get list of metrics is to request them from the prometheus port and read text response containing help comments like that:

```
> curl http://127.0.0.1:5551/metrics
# HELP blocks_height Total number of blocks in chain
# TYPE blocks_height gauge
blocks_height 135543
# HELP peers_number Total number peers to send transactions and request proposals
# TYPE peers_number gauge
peers_number 7
# HELP number_of_domains Total number of domains in WSV
# TYPE number_of_domains gauge
number_of_domains 14
# HELP total_number_of_transactions Total number of transactions in blockchain
# TYPE total_number_of_transactions gauge
total_number_of_transactions 216499
# HELP number_of_signatures_in_last_block Number of signatures in last block
# TYPE number_of_signatures_in_last_block gauge
number_of_signatures_in_last_block 5
```

It would be convenient to add and maintaing list of all metrics in iroha readthedocs' documentation

### PR DESCRIPTION
**Pull request by LiraLemur**
Signed-off-by: Sara <lira.lemur@gmail.com>

### Description of the Change

General information about metrics

### Benefits

Metrics in docs

### Possible Drawbacks

Not very full info. Yet.

Closes #1280

### CODE REVIEW

File: docs/source/maintenance/metrics.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+To use metrics, you need to add it to your `Iroha configuration <../configure/index.html#deployment-specific-parameters>`_ and use Iroha version that is newer than 1.2.1.
+
+Then, you can simply use the ip address to access the data from the running Iroha instance.
+
+Here is an example:
+
+.. code-block:: shell
+
+  > curl http://127.0.0.1:8080/metrics
```
Comment (kuvaldini) [inline docs/source/maintenance/metrics.rst:20]: should not be in code block

File: docs/source/maintenance/metrics.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  	will give you results like:
```
Comment (kuvaldini) [inline docs/source/maintenance/metrics.rst:22]: Should not be in code-block

File: docs/source/maintenance/metrics.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Here is an example:
+
+.. code-block:: shell
+
+  > curl http://127.0.0.1:8080/metrics
+
+  	will give you results like: 
+
+
+	# HELP blocks_height Total number of blocks in chain
```
Comment (kuvaldini) [inline docs/source/maintenance/metrics.rst:25]: please remove redundant identation.

hint: you can edit it on GitHub with instant preview

**Review by kuvaldini — CHANGES_REQUESTED** [review]
.

File: docs/source/concepts_architecture/consensus.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
+=====================
+Yet Another Consensus
+=====================
+
+The heart of any blockchain framework is its consensus - an algorithm that decides how the nodes agree on a value.
+In HL Iroha such algorithm is called YAC - Yet Another Consensus. 
+
+Important Concepts
```
[the remark this answers is not in the document]
Comment (LiraLemur) [reply docs/source/concepts_architecture/consensus.rst:8]: Omg, thanks for noticing. Apparently restoring the branch for the review fox somehow added it there with those were unstaged changes. I'll delete it

[the remark this answers is not in the document]
Comment (LiraLemur) [reply docs/source/concepts_architecture/consensus.rst]: @kuvaldini should be removed now, the rest seems to be fixed based on the correct build.

**Review by kuvaldini — APPROVED** [review]

**Review by kamilsa — APPROVED** [review]

**Pull request merged**